packages/compass-assistant/src/prompts.ts

```diff
@@ -1,6 +1,16 @@
 import type { ConnectionInfo } from '@mongodb-js/connection-info';
 import { redactConnectionString } from 'mongodb-connection-string-url';
 
+export const buildConversationInstructionsPrompt = ({
+  target,
+}: {
+  target: string;
+}) => {
+  // TODO: we'll want to greatly expand on this, but at minimum this is where we
+  // make the distinction between running inside Data Explorer vs Compass.
+  return `You are an assistant running in a side-panel inside ${target}.`;
```

Comment:
I changed this to
```
  return `You are an assistant running in a side-panel inside MongoDB ${target}. Always provide instructions that is specified to MongoDB ${target} unless the user asks otherwise.`;
```
and this helped quite a bit.

Reply:
I'll incorporate that. I left out the MongoDB so we can easily experiment with including that in the name or not.